contracts/LosslessReporting.sol

```diff
@@ -175,7 +175,6 @@ contract LosslessReporting is Initializable, ContextUpgradeable, PausableUpgrade
     /// @param account Potential malicious address
     function report(address token, address account) public notBlacklisted whenNotPaused returns (uint256){
         require(account != address(0), "LSS: Cannot report zero address");
-        require(!losslessController.blacklist(account), "LSS: Already blacklisted");
```

Comment:
Could you remind me why we are removing this?

Reply:
This would prevent for an account to be reported on a different token. 

For example, if the account stole TokenA and TokenB. 

- First it gets reported on TokenA
- When getting reported for TokenB that require would revert the transaction and it'd have stolen succesfully TokenB